locals.tf

```diff
@@ -8,6 +8,7 @@ locals {
     tags                  = []
     wait_for_pool_ready   = true
     container_runtime     = "containerd"
+    public_ip_disabled    = false
```

Comment:
Because the default value of the provider is `false`, I think there is no need to add this line.
Source: https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_pool#public_ip_disabled

Reply:
Seems legit ! It needs to be tested though, i'll do that asap :)